Bot Sharder/README.md

```diff
@@ -1,112 +1,37 @@
 
+**NOTE: Do not use this unless your bot is in over 2000 guilds, there is no point if its below, it uses more resources, and will probally break commands that you already use**
+
 # DBM Mods Bot Sharding
 
 **Allows your bot created by Discord Bot Maker to take advantage of sharding!**
 
+
 Installation
 ====
-Download the ZIP and extract the files to your Bot Base Folder.
+Download the [ZIP file](https://downgit.github.io/#/home?url=https://github.com/dbm-network/custom-files/blob/master/Bot%20Sharder/sharded-bot.js) and extract the file to your bot folder.
 
 Your Bot folder should look like this:
 
-![shard](https://i.gyazo.com/97ac758cb15b87be9df98019d42fc3b6.png)
+![shard](https://i.imgur.com/sHqbJjV.png)
 
 
 Running
 ====
 
 Once those are in the folder, up at the top where the path is, select everything in there and type cmd and press enter:
-![cmd](https://i.gyazo.com/9b70a9d74141836672e34106d37d1a61.png)
-
-
+![cmd](https://i.imgur.com/XWKxwob.png)
 
-In the command window that opens, type `node sharded-bot.js`
-![node](https://i.gyazo.com/a2f31037eb4dd6623da6bbd5a88a236e.png)
 
 
-If you want to run it with a linux terminal then instead of running it with `bot node.js` you run it `node sharded-bot.js`
+In the Command Prompt window that opens, type `node sharded-bot.js`. It should look somewhat similar to this.
+![node](https://i.imgur.com/AKuzOrR.png)
 
 
+If you are running on linux replace `node bot.js` with `node sharded-bot.js`.
```

Comment:
The last phrase doesn't make sense. However, since their bot is, in theory, in over 2000 guilds they should already know how to run it using cmd, so the part above telling them how to open cmd can probably be removed, and this part can be simplified